lang/pypy/Portfile

```diff
@@ -244,26 +264,27 @@ post-patch {
 # intensive, and one of the best cases for using PyPy itself over
 # CPython.
 if {$subport ne $name} {
-    # just use our own pypy2 for building pypy3
+    # just use our own pypy3.x for building other pypy3.x subports
     depends_build-append port:pypy
-    build.cmd       ${prefix}/lib/pypy/bin/pypy
+    build.cmd       ${prefix}/lib/pypy/bin/pypy-${python.branch}
 } else {
     # use a binary distribution of pypy itself
     checksums-prepend       ${distname}${extract.suffix}
 
     if {${os.major} >= 20} {
         # older versions cannot import C extensions on macOS 11 and later
-        set bootstrapper        "pypy2.7-v${version}-osx64"
-        distfiles-append        ${bootstrapper}${extract.suffix}
+        set bootstrapper        "pypy2.7-v7.3.20-macos_arm64"
```

Comment:
Presumably this file contains only arm64 binary code which will fail on x86_64 machines.

Reply:
It seems that is a wider project policy decision. Upstream only supports python 3.11 and 2.7 (needed for translation) and yet this recipe is producing older versions of python3

Reply:
Are you saying pypy only runs on one version of 3.x? Was this the case before the migration too?

Reply:
Currently we support only python3.11. You can see what versions are supported on what dates by looking at our [binary downloads](https://downloads.python.org/pypy/). The last version to support python3.10 was released in Feb 2025. the last to support python3.9 was in April 2024, the last to support python3.8 was Dec 2022.